apps/vr-tests/src/stories/Popover.stories.tsx

```diff
@@ -0,0 +1,53 @@
+import { storiesOf } from '@storybook/react';
+import * as React from 'react';
+import Screener from 'screener-storybook/src/screener';
+import { Popover, PopoverTrigger, PopoverSurface, PopoverProps } from '@fluentui/react-popover';
+
+const ExampleContent = () => {
+  return (
+    <div>
+      <h3>Popover content</h3>
+
+      <div>This is some popover content</div>
+    </div>
+  );
+};
```

Comment:
It'd be nice to give this a border so it's easier to review the alignment in the screener results.

Reply:
That's a good idea I'll do that 👍